src/components/Chat/AssistantSubModelSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import { OpenAIEntityModel } from '@/src/types/openai';
+
+import { useAppSelector } from '@/src/store/hooks';
+import { ModelsSelectors } from '@/src/store/models/models.reducers';
+
+import { Combobox } from '../Common/Combobox';
+import { ModelSelectRow, SettingContainer } from './ConversationSettings';
+
+interface Props {
+  assistantModelId: string;
+  onSelectAssistantSubModel: (modelId: string) => void;
+}
+
+export const AssistantSubModelSelector = ({
+  assistantModelId,
+  onSelectAssistantSubModel,
+}: Props) => {
+  const { t } = useTranslation('chat');
+  const modelsModels = useAppSelector(ModelsSelectors.selectModelsOnly);
```

Comment:
modelsModels? i think onlyModels will be better :)

Reply:
Thank you for suggestion.